dev/bots/test.dart

```diff
@@ -435,18 +435,34 @@ Future<void> _runTests() async {
 }
 
 Future<void> _runWebTests() async {
-  await _runFlutterWebTest(path.join(flutterRoot, 'packages', 'flutter'), tests: <String>[
-    'test/foundation/',
-    'test/physics/',
-    'test/services/',
-    // TODO(yjbanov): re-enable when flakiness is resolved
-    // 'test/rendering/',
-    // 'test/painting/',
-    // 'test/scheduler/',
-    // 'test/semantics/',
-    // 'test/widgets/',
-    // 'test/material/',
-  ]);
+  final Directory flutterPackageDir = Directory(path.join(flutterRoot, 'packages', 'flutter'));
+  final Directory testDir = Directory(path.join(flutterPackageDir.path, 'test'));
+
+  final List<String> directories = <String>[];
+
+  // TODO(yjbanov): we're getting rid of this blacklist as part of https://github.com/flutter/flutter/projects/60
+  const List<String> kBlacklist = <String>[
+    'test/cupertino',
+    'test/examples',
+    'test/material',
+    'test/painting',
+    'test/rendering',
+    'test/semantics',
+    'test/widgets',
+  ];
+
+  for (Directory subDirectory in testDir.listSync().whereType<Directory>()) {
```

Comment:
Looks like this can be done with a `.map()` and `.where()`. But if you prefer the imperative style, go for it :)

Reply:
Done.